pkg/operator/storageclasscontroller/storageclasscontroller.go

```diff
@@ -83,6 +84,9 @@ func NewStorageClassController(
 		secretInformer.Informer(),
 		infraInformer.Informer(),
 		operatorClient.Informer(),
+		// This informer isn't used anywhere else in the code, but it's added here
+		// to make this controller resyncs when users change StorageClasses.
+		scInformer.Informer(),
 	).WithSync(
```

Comment:
We could increase the resync interval to 20 minutes, now that we are watching storage classes too.

Reply:
I was going to do that initially, but I left as is because both CSI controllers resync every minute.

Reply:
Set to 20 min.